packages/nimble-components/src/table/specs/table-column-placeholder-hld.md

```diff
@@ -0,0 +1,200 @@
+-   If relevant to your data source, make sure to consider the IEEE 754 special cases of `-Inf`, `+Inf`, `-0`, `+0`, and `NaN`.
+
+#### Date column
+
+| Special-cased field values          | Cell display                                                 | Group row display  |
+| ----------------------------------- | ------------------------------------------------------------ | ------------------ |
+| `undefined`                         | column placeholder, or empty if no placeholder is configured | `"No value"`       |
+| `null`                              | column placeholder, or empty if no placeholder is configured | `"No value"`       |
+| Invalid value (e.g. `Number.NaN`)\* | \<empty cell>                                                | \<empty group row> |
+
+\*This is considered invalid data from the table's perspective and should be fixed within the client application.
+
+Column best practices:
+
+-   Avoid mixing `undefined` and `null` as values for the same field. When grouping this will lead to two groups (one for `null` values and one for `undefined` values) that both have the text `"No value"`.
+
+#### Duration column
+
+| Special-cased field values          | Cell display                                                 | Group row display  |
+| ----------------------------------- | ------------------------------------------------------------ | ------------------ |
+| `undefined`                         | column placeholder, or empty if no placeholder is configured | `"No value"`       |
+| `null`                              | column placeholder, or empty if no placeholder is configured | `"No value"`       |
+| Invalid value (e.g. `Number.NaN`)\* | \<empty cell>                                                | \<empty group row> |
+
+\*This is considered invalid data from the table's perspective and should be fixed within the client application.
+
+Column best practices:
+
+-   Avoid mixing `undefined` and `null` as values for the same field. When grouping this will lead to two groups (one for `null` values and one for `undefined` values) that both have the text `"No value"`.
+
+#### Icon mapping column
+
+The icon mapping column will not have a configuration for a placeholder.
+
+| Special-cased field values | Cell display  | Group row display  |
+| -------------------------- | ------------- | ------------------ |
+| `undefined`                | \<empty cell> | `"No value"`       |
+| `null`                     | \<empty cell> | `"No value"`       |
+| Non-mapped value\*         | \<empty cell> | \<empty group row> |
+
+\*This is considered invalid data from the table's perspective and should be fixed within the client application.
+
+In the future we can add the ability to create a mapping to a blank icon. This will allow a column to specify that it doesn't want an icon to be displayed in a cell for certain record values, but the column can still display an application-specified label on group rows.
```

Comment:
Since #1874 is complete could you update this to be a current best practice and also delete the Future Work section below?